demos/blockfactory/index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="target-densitydpi=device-dpi, height=660, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
   <title>Blockly Demo: Blockly Developer Tools</title>
-  <script src="../../blockly_compressed.js"></script>
```

Comment:
Oh, missed this. Please upload a rebuilt compressed file instead of swapping it.

Reply:
Gah I meant to do that!

Reply:
So the new field_label_serializable isn't required by any other scripts at the moment, which I think is making it so closure doesn't include it in the compressed file. Should I try to [export ](https://developers.google.com/closure/compiler/docs/api-tutorial3#mixed)it? or is there a different solution?

Reply:
You need to add a goog.require for the new field type to blockly.js.

Side note: You can see a require for FieldDate commented out in that file which is what keeps it from being included in Blockly by default since the date picker is a relatively large dependency.

Reply:
Rebuilt and retested!